crates/codegen/runtime/cargo/wasm/src/runtime/wrappers/ast/generated/selectors.rs

```diff
@@ -83,15 +83,15 @@ impl Helper {
             let Edge { label, node } = self.node.children.get(self.index)?;
 
             match label {
-                // Skip unlabeled nodes:
-                | None
+                // Skip root nodes:
+                | EdgeLabel::Root
```

Comment:
So, IIUC, the only way a `Root` could exist as a child, is if a user manually created such a tree, since we will never create it ourselves.

In this corner case, WDYT of not skipping `Root`? that would be less confusing to the user.